graph/src/data_source/subgraph.rs

```diff
@@ -224,7 +282,38 @@ impl UnresolvedDataSource {
             "source" => format_args!("{:?}", &self.source),
         );
 
-        let kind = self.kind;
+        let kind = self.kind.clone();
+        let source_manifest = self.resolve_source_manifest::<C>(resolver, logger).await?;
+        let source_spec_version = &source_manifest.spec_version;
+
+        if source_spec_version < &SPEC_VERSION_1_3_0 {
+            return Err(anyhow!(
+                "Source subgraph manifest spec version {} is not supported, minimum supported version is {}",
+                source_spec_version,
+                SPEC_VERSION_1_3_0
+            ));
+        }
+
+        let pruning_enabled = match source_manifest.indexer_hints.as_ref() {
```

Comment:
pruning can also be set outside of the manifest right? Is there are a way we check that too?